site-src/implementations.md

```diff
@@ -37,7 +37,7 @@ cover, and documentation to help users get started.
 - [NGINX Gateway Fabric][12] (GA)
 - [ngrok][33] (preview)
 - [STUNner][21] (beta)
-- [Traefik][13] (alpha)
+- [Traefik Proxy][13] (GA)
```

Comment:
You also need to change the line that this alias points to for the link to work (I think line ~70). See the deploy preview here: https://deploy-preview-3182--kubernetes-sigs-gateway-api.netlify.app/implementations/

Reply:
Good catch, I fixed the link.